code/index.html

```diff
@@ -2,29 +2,330 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Escape House</title>
+    <script
+      src="https://kit.fontawesome.com/7ed1b41d25.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header class="header">
+      <div class="header__title">
+        <h1>ESCAPE HOUSE</h1>
+        <a class="header__title__arrow" href="#top"
+          ><i class="fas fa-chevron-down"></i
+        ></a>
+      </div>
+      <video
+        class="header__video header__video--mobile"
+        autoplay
+        muted
+        playsinline
+        loop
+      >
+        <source src="./videos/header-video-iphone.mp4" />
+      </video>
+      <video
+        class="header__video header__video--ipad"
+        autoplay
+        muted
+        loop
+        playsinline
+      >
+        <source src="./videos/header-video-ipad.mp4" />
+      </video>
+      <video
+        class="header__video header__video--desktop"
+        autoplay
+        muted
+        loop
+        playsinline
+      >
+        <source src="./videos/header-video-desktop.mp4" />
+      </video>
+    </header>
 
-    <!-- Signup form -->
+    <div id="top"></div>
+    <section class="navbar">
+      <a href="#top">
+        <img
+          class="navbar__logotype"
```

Comment:
I used to do it, but it is a part of the BEM method that I was trying out (the black step of the assignment) to use 2 _. Since wanted to try that out I didnt use the  -  = ).